### DISCUSSION TITLE
Missing file after update donet 8

### DISCUSSION THREAD

**Discussion by arunkumar12712** [serenity-is/Serenity#7096] [General]
<img width="645" alt="Upload" src="https://github.com/serenity-is/Serenity/assets/15244940/bae0b553-aa08-4f57-b6ed-1cdfbd187b2d">
<img width="531" alt="image" src="https://github.com/serenity-is/Serenity/assets/15244940/553ed568-e2cd-45e9-905f-0bd9e9feefd4">

**Comment by VictorTomaili** [comment]
Did you read the release notes?

https://serenity.is/docs/release-notes/8.2.1
